VocaDb.ResXFileCodeGenerator/VocaDb.ResXFileCodeGenerator.csproj

```diff
@@ -15,7 +15,7 @@
     <CopyLocalLockFileAssemblies>true</CopyLocalLockFileAssemblies>
     <NoWarn>$(NoWarn);NU5128</NoWarn>
     <DevelopmentDependency>true</DevelopmentDependency>
-    <Version>2.0.2</Version>
+    <Version>3.0.0</Version>
```

Comment:
If there are no breaking changes in this PR, I think bumping the version number to either `2.0.3` or `2.1.0` makes more sense.

Reply:
True, there are however substantial speed increases and the generation is based on what the editor says the file contains rather than the file content. I personally still think it should be a 3.0.
Also, I am adding the partial class generation we discussed in issue #5, that will be included in next commit.